### DISCUSSION TITLE
how to disable yazi statusline?

### DISCUSSION THREAD

**Discussion by harisonkhlil** [sxyazi/yazi#355] [1. Q&A] [answered]
<img width="1493" alt="image" src="https://github.com/sxyazi/yazi/assets/70578017/19fe5c6d-d977-438c-993b-d25acf9a88db">

If I open multiple tmux Tabs, the statusline of tmux will conflict with yazi.

**Comment by sxyazi** [comment, marked as answer]
I've added it to the tips: https://yazi-rs.github.io/docs/tips#no-status-bar

![image](https://github.com/sxyazi/yazi/assets/17523360/fc988aa8-eb0f-4810-a135-0142efd56baf)
